crates/bins/src/bin/datadog-static-analyzer-git-hook.rs

```diff
@@ -124,6 +128,13 @@ fn main() -> Result<()> {
         "484df034272a3f63c1796ef44f83a790d7729590",
     );
 
+    opts.optopt(
+        "",
```

Comment:
why not allow -o for short?

```suggestion
        "o",
```